### PR TITLE
Visual tweaks and better theming support

### PR DESCRIPTION
**Pull request by stefangabos**
#### some visual tweaks:
- wrongly displaying empty lines in source code (non-empty lines collapse over the empty ones)
- the last line of every block of prettified source code had wrong padding
- styled the @todo tag and markdown's blockquotes (> something) so that they stand out a bit
#### better theming support
- removed any styling in jsdoc.css that was overwriting the styles in prettify.css so that one can now use themes from https://jmblog.github.io/color-themes-for-google-code-prettify/ simply by replacing the content of the prettify.css file

I've attached an example file so that you can see how it looks with the updates [zebra.src.zip](https://github.com/clenemt/docdash/files/521964/zebra.src.zip)

Thanks!

### CODE REVIEW

**Comment by ar2rsawseen** [comment]
With approval from author, finally we decided to go with our own version of this repo and published it as npm package. Feel free to check it out and send your PRs there.
https://www.npmjs.com/package/countly-docdash

**Comment by ar2rsawseen** [comment]
Hello,
if you still want to give this PR a go, I will merge it, but there are some things there

so for some reason jsdoc.css file shows as binary modified, can you recheck how it does look on your end?
And why commenting out doc.name?

**Comment by stefangabos** [comment]
I don't remember why doc.name is commented out...
jsdoc.css looks find if you view it - maybe GitHub found something funny in it and considers it a binary file...

**Comment by ar2rsawseen** [comment]
Ok, I tried out locally, and still got binary, then I just copied contents of it from your fork and tried merging, but styles have changed so much since then, that it is hard to follow for me which changes are yours, and which are overwritten other people changes.
So I am closing this PR, but if you are up to it, I will gladly review and merge new PR
